### PR TITLE
[PWD-2527] Finalize My Account Company Info

### PR DESCRIPTION
**Pull request by haikal1223**
### What type of PR is this? (_check all applicable_)

- [x] Feature
- [ ] Bug Fix
- [ ] Documentation Update
- [ ] Setup
- [ ] Code Refactor
- [ ] Performance Improvements
- [ ] Test
- [ ] Build
- [ ] CI
- [ ] Revert

### CODE REVIEW

File: src/components/PWDKDropzone.tsx [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+import {
+  FC,
+  PropsWithChildren,
+  forwardRef,
+  useEffect,
+  useImperativeHandle,
+  useRef,
+  useState,
+  Children
+} from 'react'
```
Comment (Vesals) [inline src/components/PWDKDropzone.tsx:13]: tiati kal di #56 juga ada dropzone adjustment

Comment (haikal1223) [reply src/components/PWDKDropzone.tsx:13]: ini udah collab sama gilangs 👌

Comment (Vesals) [reply src/components/PWDKDropzone.tsx:13]: oke mantappp

**Review by Vesals — APPROVED** [review]

**Review by kuntiarso — APPROVED** [review]
🔆🔆

**Review by gangsararyop — APPROVED** [review]

**Pull request merged**